types/nova-editor-node/index.d.ts

```diff
@@ -1009,7 +1009,7 @@ interface Workspace {
     readonly config: Configuration;
     readonly textDocuments: ReadonlyArray<TextDocument>;
     readonly textEditors: ReadonlyArray<TextEditor>;
-    readonly activeTextEditor: TextEditor;
+    readonly activeTextEditor: TextEditor | null;
```

Comment:
Is this `undefined`, or `null`?

Reply:
Worst case scenario, I'd just suggest this just in case:

```suggestion
    readonly activeTextEditor: TextEditor | null | undefined;
```

Reply:
Here's Logan from Panic's reply:

> This is a consequence of using Apple’s JavaScriptCore and its Objective-C bridge (as Nova’s extension runtime is implemented in Objective-C, and bridged into JavaScript). It treats null and undefined as effectively the same thing in a lot of cases.
>
> We attempt to ensure it’s one or the other in a lot of cases, but most of the time if it can be null, it can also be undefined.

So it seems anything that can be null could also be undefined. What do you all think we should do? There are probably a lot of other properties in here that can be null | undefined.